python/tests/lang/test_references.py

```diff
@@ -0,0 +1,33 @@
+from unittest import TestCase
+
+from aac.lang.references import get_definition_references_from_list
+
+from tests.helpers.parsed_definitions import create_data_definition, create_data_ext_definition, create_field_entry, create_model_definition
+
+
+class TestLangReferences(TestCase):
+
+    def test_get_definition_references_from_list(self):
```

Comment:
Need to add a test to demonstrate that references won't return the source definition as a reference. In other words, the declaration of a definition is not a reference to itself.